scripts/cube_fx.lua

```diff
@@ -310,10 +368,20 @@ local function track_victory_statistics(size, results)
     end
   end
   victory_statistics.cube_last_position = new
+
+  -- Track cube "utilization"
+  local machine_conditional = cube_working_machine_types[entity.type]
+  if machine_conditional and machine_conditional(entity, results[1].item) then
```

Comment:
If there's more than 1 cube we could randomly choose the index to check for utilization to get a less biased sampling.

Could also check them all and multiply the amounts we're adding to `working`/`idle` by the `search_result_weight` of the item defined in `cube_search.lua` but that's more expensive.

Reply:
I went for choosint a random cube to keep the processing to a minimum.